node/cmd/guardiand/node.go

```diff
@@ -1456,6 +1511,42 @@ func runNode(cmd *cobra.Command, args []string) {
 		}
 	}
 
+	// A check above does not allow this in mainnet.
+	if shouldStart(arbitrumSepoliaRPC) {
+		wc := &evm.WatcherConfig{
+			NetworkID: "arbitrumsepolia",
+			ChainID:   vaa.ChainIDArbitrumSepolia,
+			Rpc:       *arbitrumSepoliaRPC,
+			Contract:  *arbitrumSepoliaContract,
+		}
+
+		watcherConfigs = append(watcherConfigs, wc)
+	}
+
+	// A check above does not allow this in mainnet.
+	if shouldStart(baseSepoliaRPC) {
```

Comment:
This is more thinking out loud, but is there a sensible way to put that logic inside "shouldStart"?